.github/workflows/snapshot.yml

```diff
@@ -55,6 +55,7 @@ jobs:
           LIBUSB1_CONFIG: --enable-shared --disable-static
           HIDAPI_CONFIG: --enable-shared --disable-static --disable-testgui
           CAPSTONE_CONFIG: "CAPSTONE_BUILD_CORE_ONLY=yes CAPSTONE_STATIC=yes CAPSTONE_SHARED=no"
+          CAPSTONE_CFLAGS: -I$(CAPSTONE_SRC)/include/capstone
```

Comment:
From github build log, capstone includes get installed in /home/runner/work/riscv-openocd/build/i686-w64-mingw32-root/usr/include/capstone
Why using the source code here instead of the installation?
Apparently the issue is in some crap commands run by github on the file capstone.pc AFTER it is installed.
From line 2042 to 2049
https://github.com/riscv/riscv-openocd/runs/6293338772?check_suite_focus=true#step:8:2042
The file BEFORE this crap works and
 pkg-config --cflags capstone
 -I/usr/include/capstone
but after the crap the same command gives no output!
No idea how to fix it

Reply:
This is definitely a hack, but all I really care about is that the Windows build passes so it'll catch problems with new code. If this is fixed properly upstream then that will come down here in time.